src/content/docs/apm/agents/java-agent/instrumentation/java-agent-instrument-kafka-message-queues.mdx

```diff
@@ -43,9 +48,21 @@ To view these metrics, create a custom dashboard:
    ```
    MessageBroker/Kafka/Internal/consumer-metrics/request-rate
    ```
+   
+   If you are using Kafka Streams, you can find streams metrics in this metric folder:
+   
+   ```
+   Kafka/Streams/KafkaStreamsMetricName
+   ```
+   
+   For example, the the `poll-latency-avg` metric is located at:
+   
+   ```
+   Kafka/Streams/stream-thread-metrics/poll-latency-avg
+   ```
 
    <Callout variant="tip">
-     For a full list of Kafka consumer and producer metrics, see the [Kafka documentation](https://kafka.apache.org/documentation/#monitoring).
+     For a full list of Kafka consumer, producer, and streams metrics, see the [Kafka documentation](https://kafka.apache.org/documentation/#monitoring).
```

Comment:
I would change this link to https://kafka.apache.org/documentation/#remote_jmx because that's where the actual table of metrics are. 

```suggestion
     For a full list of Kafka consumer, producer, and streams metrics, see the [Kafka documentation](https://kafka.apache.org/documentation/#remote_jmx.
```